src/styles/theme/helper.scss

```diff
@@ -81,3 +81,17 @@ hr {
   overflow: visible;
   text-align: inherit;
 }
+
+.oc-rounded {
+  border-radius: 5px;
+}
+
```

Comment:
What do you think about two additional classes, `oc-rounded-top` and `oc-rounded-bottom` for only setting the 5px border radius top or bottom? I guess that there are situations where you know that you only need the rounded borders top or bottom.

Reply:
And in such cases it would be nicer to have a dedicated class for it instead of applying one class for rounded corners and then another class for removing rounded corners on one end again.

Reply:
Yeah true, let me fix that in a minute